report/Titelblad.tex

```diff
@@ -33,7 +33,16 @@
   \includegraphics[width=0.9\textwidth]{images/aau_logo.pdf}\\[1.0 cm]
  \end{flushright}
  \begin{flushleft}
-  \textbf{Abstract:}\\INSERT HERE
+  \textbf{Abstract:}\\
+  In the danish education system, the Arduino platform is often used to introduce the students to computer science. 
+  The Arduino's hardware is relatively simple simple and easy to grasp, however the Arduino language can often be difficult for beginners.
+  The Arduino language is a subset of C++, and while it is simplified compared to C++, there are still parts of the language that are difficult to comprehend for novices.
+  
+  This report is meant to elaborate and explain the development of the HCL language, a programming language designed to ease the introduction to programming for students and novices.
```

Comment:
reformulate to:
This report elaborates and explains...
meant is the alias of will